samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java

```diff
@@ -79,39 +82,53 @@ public static ImportProductsRequest getImportProductsGcsRequest(
             .setErrorsConfig(errorsConfig)
             .build();
 
-    System.out.println("Import products from google cloud source request: " + importRequest);
+    System.out.printf("Import products from google cloud source request: %s%n", importRequest);
 
-    return importRequest;
-  }
-
-  public static void waitForOperationCompletion(ImportProductsRequest importRequest)
-      throws IOException, InterruptedException {
+    // Initialize client that will be used to send requests. This client only
+    // needs to be created once, and can be reused for multiple requests. After
+    // completing all of your requests, call the "close" method on the client to
+    // safely clean up any remaining background resources.
     try (ProductServiceClient serviceClient = ProductServiceClient.create()) {
       String operationName = serviceClient.importProductsCallable().call(importRequest).getName();
-      System.out.printf("OperationName = %s\n", operationName);
+
+      System.out.println("The operation was started.");
+      System.out.printf("OperationName = %s%n", operationName);
 
       OperationsClient operationsClient = serviceClient.getOperationsClient();
       Operation operation = operationsClient.getOperation(operationName);
 
-      while (!operation.getDone()) {
-        // Keep polling the operation periodically until the import task is done.
-        Thread.sleep(30_000);
+      long assuredBreak = System.currentTimeMillis() + 60000; // 60 seconds delay
```

Comment:
In PR #481 we're doing this (maybe we can use the same style)?
        Instant deadline = Instant.now().plusSeconds(60);

        while (!operation.getDone() || Instant.now().isBefore(deadline)) {

Reply:
Changed it accordingly.